### PR TITLE
Add import/export for tags and projects

### PR DESCRIPTION
**Pull request by chriswk**
- Also adds import/export for tag-types
- As well as feature_tags, since the reference is by name
  and not by generated serial, it should be easy to export between
  instances

### CODE REVIEW

File: src/lib/db/feature-toggle-store.js [hunk 1]
```diff
@@ -260,6 +260,37 @@ class FeatureToggleStore {
...
 
+    async getAllFeatureTags() {
+        const rows = await this.db(FEATURE_TAG_TABLE).select(
+            FEATURE_TAG_COLUMNS,
+        );
+        return rows.map(row => {
+            return {
+                featureName: row.feature_name,
+                tagType: row.tag_type,
+                tagValue: row.tag_value,
```
Comment (ivarconr) [inline src/lib/db/feature-toggle-store.js:271]: not relevant for this PR, but it really bugs me that we end up doing snake_case to camelCase conversions all the time.

Comment (ivarconr) [reply src/lib/db/feature-toggle-store.js:271]: Maybe https://www.npmjs.com/package/camelcase is an option?

Comment (ivarconr) [reply src/lib/db/feature-toggle-store.js:271]: https://github.com/sindresorhus/camelcase-keys

**Review by ivarconr — APPROVED** [review]
yet again a solid PR. Can't spot anything special to fix.

File: src/lib/db/feature-toggle-store.js (hunk 1 above)
Comment (chriswk) [reply src/lib/db/feature-toggle-store.js:271]: I'm wondering if having proper types would help, could state that all stores implement a column name <~> jsonName translator so snake_case is internal to store while we use camelCase for arguments to- and return objects from store methods?

Comment (chriswk) [reply src/lib/db/feature-toggle-store.js:271]: Though this PR is probably not the place to fix it. :)

**Pull request merged**